weather.js

```diff
@@ -0,0 +1,36 @@
+var city= "Beograd"
+
+function myFunction() {
+     city = document.getElementById("myText").value;
+    console.log(city);
+
+    checkWeather();
+  }
+
+
+const apiKey = "ffcc3b1d6839acddfd18b10b824cdae1";
+const apiUrl=  "https://api.openweathermap.org/data/2.5/weather?&units=metric&q=";
+
+async function checkWeather(){
+    const response = await fetch(apiUrl + city +'&appid=' + apiKey);
+    var data = await response.json();
+    
+    document.querySelector("#demo").innerHTML = city;
+    document.querySelector(".temp").innerHTML = data.main.temp + " °C";
+    document.querySelector(".humidity").innerHTML = "H: " + data.main.humidity + " %";
+    document.querySelector(".pressure").innerHTML = "P: " + data.main.pressure + " Ph";
+    document.querySelector(".wind").innerHTML =  "WS: " + data.wind.speed + " KM/H";
+    if(data.clouds.all > 39 && data.clouds.all < 70){
+        document.getElementById("slika").src= "semi.png";
+    }
+    if(data.clouds.all > 70){
+        document.getElementById("slika").src= "cloudy.png";
+    }
+    if(data.weather === "Rainy"){
```

Comment:
data.weather is object in order to get weather condition you need to access object property
object example:
{
    "id": 800,
    "main": "Clear",
    "description": "clear sky",
    "icon": "01d"
}
So you need to change data.weather -> data.weather[0].main

Reply:
Changed thank you